### PR TITLE
[FIX] l10n_lt: tag (missing) accounts for financial reports

### PR DESCRIPTION
**Pull request by svfu-odoo**
In the related enterprise PR the LT financial reports (BS and P&L) were converted to use account tags instead of account codes. This commit ensures that all accounts are tagged and thus appear in the P&L or BS.

task-3701151

### CODE REVIEW

**Review by rugo-odoo — APPROVED** [review]
LGTM (cc @william-andre)

File: addons/l10n_lt/i18n_extra/l10n_lt.pot [hunk 1]
```diff
@@ -6,9 +6,9 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 12.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-09-05 09:02+0000\n"
-"PO-Revision-Date: 2019-09-05 09:02+0000\n"
-"Last-Translator: <>\n"
+"POT-Creation-Date: 2024-02-15 15:33+0000\n"
```
[the remark this answers is not in the document]
Comment (svfu-odoo) [reply addons/l10n_lt/i18n_extra/l10n_lt.pot]: I removed it

File: addons/l10n_lt/models/template_lt.py [hunk 2]
```diff
@@ -37,3 +37,15 @@ def _get_lt_res_company(self):
...
+        if template_code == "lt":
+            bank_tags = self.env.ref('l10n_lt.account_account_tag_b_4')
+            company.account_journal_suspense_account_id.tag_ids = bank_tags
+            company.account_journal_payment_debit_account_id.tag_ids = bank_tags
+            company.account_journal_payment_credit_account_id.tag_ids = bank_tags
+            company.transfer_account_id.tag_ids = bank_tags
+
+            other_operating_results_tags = self.env.ref('l10n_lt.account_account_tag_6_other_operating_results')
+            company.default_cash_difference_income_account_id.tag_ids = other_operating_results_tags
+            company.default_cash_difference_expense_account_id.tag_ids = other_operating_results_tags
```
[the remark this answers is not in the document]
Comment (svfu-odoo) [reply addons/l10n_lt/models/template_lt.py]: I just added the tags now instead of overwriting them

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**